cmd/cli/triggers.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"time"
+
+	"github.com/moira-alert/moira"
+)
+
+func handleRemoveTriggersByPrefix(logger moira.Logger, database moira.Database, prefix string) error {
+	triggers, err := database.GetTriggerIDsByPrefix(prefix)
+	if err != nil {
+		return err
+	}
+
+	// Added delay because command is potentially dangerous and can delete unwanted triggers
+	const delay = 10 * time.Second
+	logger.Infof("%d triggers by prefix %s would be removed after %s", len(triggers), prefix, delay)
+	logger.Info("You can cancel execution by Ctrl+C")
+	time.Sleep(delay)
+
+	logger.Infof("Removing triggers by prefix %s started", prefix)
+	for _, id := range triggers {
+		err := database.RemoveTrigger(id)
+		if err != nil {
+			return err
```

Comment:
maybe better wrap err?

Reply:
yes, I will do, because there are 2 `return err` cases

Reply:
added both, [first ](https://github.com/moira-alert/moira/pull/790/commits/de53e6d0e9653797804aaec24236aa1765c25897#diff-bbe74c5f7c7c4e46d51dec5dfa5561be8b5fd6f26ae4ea6eac8bc27876f7d98bR13) and [second](https://github.com/moira-alert/moira/pull/790/commits/de53e6d0e9653797804aaec24236aa1765c25897#diff-bbe74c5f7c7c4e46d51dec5dfa5561be8b5fd6f26ae4ea6eac8bc27876f7d98bR26)